i18n-page-example.html

```diff
@@ -0,0 +1,57 @@
+<link rel="import" href="i18n-msg-behavior.html">
+
+<!--
+`<i18n-page-example>` Example of i18n-msg-behavior usage in a component page.
+@demo demo/page.html
+-->
+
+<dom-module id="i18n-page-example">
+  <template>
+    <h1>[[i18n.days]]</h1>
+    <span>[[i18n.hours]]</span>
+  </template>
+</dom-module>
+
+<script>
+(function () {
+
+  window.I18nMsg = window.I18nMsg || {
```

Comment:
If you put this at the start of the `i18n-msg-behavior` then it will not need to be added to every component.